ciao-controller/api/api.go

```diff
@@ -95,6 +95,7 @@ type RequestedVolume struct {
 	Description string `json:"description,omitempty"`
 	Name        string `json:"name,omitempty"`
 	ImageRef    string `json:"imageRef,omitempty"`
+	Internal    bool   `json:"-"`
```

Comment:
Does this allow clients to create internal volumes via the API?  Is this something that they can currently do?